### DISCUSSION TITLE
Chansort für Grundig

### DISCUSSION THREAD

**Discussion by Laie-absolut** [PredatH0r/ChanSort#215] [Q&A]
Kann ich mit Chansort Grundig TV bearbeiten ? Danke für alle nützlichen Antworten.

**Comment by PredatH0r** [comment]
Derzeit nicht. Ich habe erst eine einzige Grundig Senderliste gesehen, aber ich gehe davon aus, dass es auch bei diesem Hersteller (wie bei fast allen anderen) mehrere Formate oder Varianten gibt. Eine allgemeine Aussage, ob ein Hersteller unterstützt wird, ist somit ohnehin nicht möglich.
Du kannst mir gerne deine Senderliste zukommen lassen, und ein paar Infos bzgl. deines TV-Modells. Danach kann ich mehr sagen.

**Comment by PredatH0r** [comment]
Ich hab jetzt aus den 2 Files, die mir zur Verfügung standen, soweit wie möglich die Unterstützung für Grundig "dvb\*_config.xml"-Senderlisten eingebaut: https://github.com/PredatH0r/ChanSort/releases/tag/v2021-01-02
